packages/grant-explorer/src/features/round/ViewCartPage/PayoutTokenDropdown.tsx

```diff
@@ -0,0 +1,97 @@
+import { PayoutToken } from "../../api/types";
+import { Listbox, Transition } from "@headlessui/react";
+import { PayoutTokenButton } from "./PayoutTokenButton";
+import React, { Fragment } from "react";
+import { CheckIcon } from "@heroicons/react/24/solid";
+import { classNames } from "common";
+
+export function PayoutTokenDropdown(props: {
+  payoutTokenOptions: PayoutToken[];
+  selectedPayoutToken?: PayoutToken;
+  setSelectedPayoutToken: (payoutToken: PayoutToken) => void;
```

Comment:
`setSelectedPayoutToken` → `onSelectedPayoutToken` as above, to avoid leaking one component's implementation into another.

Reply:
Noted, I will leave for now to reduce changes to already working checkout code, but will make sure to fix in future versions.